File: app/src/main/java/com/vsa/filmoteca/repository/ws/WSInterface.java
package com.vsa.filmoteca.repository.ws;


import retrofit2.http.GET;
import retrofit2.http.Url;
import rx.Observable;

/**
 * Created by albertovecinasanchez on 8/3/16.
 */
public interface WSInterface {

    @GET("/es/webs-municipales/filmoteca/agenda/folder_listing")
    Observable<String> moviesListHtml();

    @GET
    Observable<String> movieDetail(@Url String url);

}

File: app/src/main/java/com/vsa/filmoteca/data/repository/ws/WsInterface.kt
package com.vsa.filmoteca.data.repository.ws

import retrofit2.Call
import retrofit2.http.POST
import retrofit2.http.Query

/**
 * Created by Alberto Vecina Sánchez on 2019-06-05.
 */
interface WsInterface {

    @POST("registrationId")
    fun registerPusHNotificationToken(@Query("token") token: String): Call<Void>

}